scripts/lst-rfperformance.py

```diff
@@ -164,6 +161,6 @@
     plot_dl2.plot_importances(reg_disp_vector, features)
     plt.show()
 
-    plt.hist(dl2[dl2['hadroness']==1]['gammaness'], bins=100)
-    plt.hist(dl2[dl2['hadroness']==0]['gammaness'], bins=100)
+    plt.hist(dl2[dl2['mc_type']==101]['gammaness'], bins=100)
```

Comment:
101?

Reply:
101 is the particle number for a hadron

Reply:
in simtelarray?

Reply:
Yes. The numbering goes as follow (actually it might even be from CORSIKA 🤔):
MC shower primary ID:
- 0 (gamma)
- 1(e-)
- 2(mu-)
- 100*A+Z for nucleons and nuclei
- negative for antimatter

Reply:
ok, maybe it is right, but according to the corsika user guide:
https://web.ikp.kit.edu/corsika/usersguide/usersguide.pdf

![image](https://user-images.githubusercontent.com/17042312/57284785-27755000-70b2-11e9-8469-2d38e59c5b49.png)

This only works for A > 2. Maybe this works, I had always used 14 for protons (and the numbers you are quoting there are different from those from CORSIKA user guide).

Anyways, this looks good, thanks @vuillaut!

Reply:
Ho ok so it does come from CORSIKA but from simtel (I just checked with a proton file and pyeventio and the shower primary id is 101).
Thank you for the info.